### ISSUE TITLE
<Email Copy Review> from <Benefits Dependent Experience>

### ISSUE THREAD

**Issue by steele-lm** [department-of-veterans-affairs/va.gov-team#65222]
## About your team

- Team name: Benefits Dependent Experience
- OCTO-DE product owner: Emily Theis
- Product manager: Laura Steele
- Designer: Ajia Wallace and Julie Pedtke
- Accessibility specialist: non
- Content writer: 
- Slack channel: benefits-ce-dependents_exp
 
### Describe what you need in a few sentences:
- We are sending an email to individuals reminding them to verify and update the dependents on their benefits by visiting their "view dependents" page on VA.gov.

### Which are you doing?
- [x] **Sending an email**

### Types of Support
What areas does your team need support in? Check all that apply.
#### Content
- [x] Collaboration on content written by your team’s dedicated content writer 
- [ ] Collaboration on content if your team does not have a dedicated content writer
#### Accessibility
- [x] Accessibility review of email design
- [ ] Accessibility issue (general request)
- [ ] Accessibility PDF review request
- [ ] Accessibility user research support ([submit a separate, additional request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name))
- [ ] DST component or pattern 
- [ ] Something else 
#### IA
- [ ] User Flows
- [ ] Page placement
- [ ] URLs
- [ ] Breadcrumbs and Left Nav placement
- [ ] Redirects
#### Multilingual content
- [ ] Translating content
 
## Timeframe
_We guide and work alongside teams across nearly every OCTO digital product. We also work with partners across VA to lead content migration and manage all unauthenticated content on VA.gov. And we're always working to expand the VA.gov content style guide and our IA and accessibility documentation. We need to prioritize all intake requests based on our overall workload and VA and OCTO priorities._

_Tell us about your product's timeline. And we'll work with you to meet timeline needs as best we can._

### Have you worked with CAIA before? 
  - [x] Yes
  - [ ] No

### What's your team’s next step and its timing?

- **Our next steps are** to determine the distribution list and build the email in a VA Notify template.
- **Provide date if available**: next 1-2 weeks

#### Timing for your next step: 
  - [ ] Next week
  - [x] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Is this timing related to a specific event or Congressionally mandated deadline?
  - [ ] Yes
  - [x] No

### If you're conducting research, when is that starting? 
**Provide date if available:** xx/xx/xxxx
  - [ ] Next week
  - [ ] This month
  - [ ] Next month
  - [ ] This quarter
  - [ ] Next quarter
  - [ ] 6 months
  - [ ] 1 year

### Will you release this new product incrementally (for example, release to 25% of users to start)? 
  - [x] Yes
  - [ ] No
 
**Note**: _If you checked yes, we’ll reach out to discuss details about the content in the react widget (we use these widgets to display entry points for new products to a certain percentage of users who visit our static pages)._

  
### When do you expect to launch your product to 100% of users?
  
*Please provide an estimated date so our team can create other relevant tickets.*
- **We expect to launch our product to 100% of users** by the end of October
  
### Will this work be going through the Collaboration Cycle?
- [ ] Yes 
- [x] No

### If no ...
- Skip to the _supporting artifacts_ section, below.
 
#### Please provide the link to your Collaboration Cycle ticket: 
- **Ticket (number)**: 
- **Collab. Cycle Ticket Opened (date)**: xx/xx/xxxx
 
#### Please check all of the phases in the Collaboration Cycle you have completed: 
- [ ] Design Intent
- [ ] Midpoint Review
- [ ] Staging
 
##### Design Intent
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Midpoint
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
##### Staging
- [ ] Accessibility
- [ ] Content
- [ ] IA
- [ ] Translation
- [ ] Already completed, no assistance needed
- [ ] Unknown at this time
  
## Supporting artifacts
*Please provide links to supporting artifacts as available.*
- **[Product outline](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/ebenefits/dependency-verification/README.md)**
- **[User flows](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1689863079145/5b97ba9ea11077f983f3413167f6324f11aa04a8?sender=u0e8ac1d6d7681bb7e1b80558)**
- **[Prototype or mockups](https://www.sketch.com/s/149b2ccf-af21-4c71-9ca8-2ef2ae8af9fc)**
- **[Content source of truth](https://docs.google.com/document/d/1iSO7jTqeA2wM-7S2sYqjt6sFoJp7h_AtEIaSz_dqTYQ/edit#heading=h.zptou6y925s)**
- **Research plan:** none
- **Any epics/issues that may be helpful**: [Dependency Verification](https://github.com/department-of-veterans-affairs/va.gov-team/issues/62218)
- **Link to testing accounts** (please do not paste them in this ticket): 

### Have you already worked with someone from OCTO leadership?
- [x] Emily Theis
- [x] Julie Strothman

### How do you plan to work with us on this request?
- [ ] I’ll take this to Beth or Danielle’s office hours.
- [ ] I’d like to schedule time to talk about this.
- [x] Let’s work async and meet as needed.
- [ ] Schedule a kick-off call with your team and CAIA for your project **(Recommended)**

 
### Accessibility Help in Slack
The [#accessibility-help channel](https://dsva.slack.com/archives/C8E985R32) in the DSVA Slack, is also available to you as a resource. Everyone is welcome to ask questions or get help from our a11y (accessibility) specialists. Tag `@Terry Nichols` to get a11y help asap.

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag `@Terry Nichols`.**

- If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
- If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)
 
 - If you are creating an [experimental design,](https://design.va.gov/about/contributing-to-the-design-system/experimental-components-and-patterns#what-is-an-experimental-component-or-pattern?) please [also contact the DST. ](https://design.va.gov/about/contributing-to-the-design-system/suggest-an-addition-or-update)

- If you are requesting [a11y support for user research,](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=coforma-terry%2Cangelafowler82&labels=508%2FAccessibility%2C+sitewide+CAIA%2C+sitewide+accessibility%2C+CAIA-a11y-research&template=caia-a11y-research.yaml&title=%5BCAIA+A11y+Research%5D%3A+Team+name%2C+product+name) please complete this form, next.

**Comment by laurwill** [comment]
Hi @juliepedtke ! We're getting you an estimate — in the meantime, could you confirm which version of the email content we should be reviewing, the content source of truth or the Sketch mockup? 
I saw some significant differences between the two. Thank you!

**Comment by juliepedtke** [comment]
Yes, review should be of the Sketch [Option 1 Updated 9/7](https://www.sketch.com/s/149b2ccf-af21-4c71-9ca8-2ef2ae8af9fc/a/9lRw3gW), not the versions on the "Archive" page. The content source of truth is incorrectly named, it was actually an earlier working doc - sorry about that!

**Comment by laurwill** [comment]
No worries, thanks for confirming!

**Comment by laurwill** [comment]
Hi @juliepedtke , sorry for delay here.

For accessibility, the main note was to provide a plain text version of the email — do you know if that functionality is included in the VA Notify template?

And here are 2 content Qs:
- **Question:** Is this flow specific to dependents listed on **disability** benefits?
- **Question:** Can you confirm where the 2 links in this email go? Are these both directing to the static ["View or change dependents" page](https://www.va.gov/view-change-dependents/), or to authenticated interactive pages to review or update dependents?

We'll have full suggested content updates to the email content for you tomorrow, just want to confirm link text first!

**Comment by juliepedtke** [comment]
Hi @laurwill ! 

I'll have to check with Notify, it's their email template so I assume there is a plain text option.

Content Qs:

- This would apply to dependents on disability and pension (all compensation) 
- This bring sup a good point, originally we'd imagined that the first link would go to the [View dependents tool](va.gov/view-change-dependents/view) and the second would go to the [686c form](https://staging.va.gov/view-change-dependents/add-remove-form-21-686c/introduction). But that would mean both would require signing in. Since some folks receiving the notification may have never signed into VA.gov, the more general landing page might be a better option?

**Comment by juliepedtke** [comment]
cc @ajialeilani

**Comment by juliepedtke** [comment]
Update from Notify: Yes, the emails do convert to plain text. There is no separate version that we need to create.

**Comment by laurwill** [comment]
Thanks for confirming @juliepedtke ! Yes, I think in this case we'd probably want to direct to the static page that explains how to review or change dependents, and then links into the tools to do either thing. 

Another benefit of linking to the static page is that it offers the other ways to add dependents like mail, etc. 

Is there another way to review and confirm your dependents, other than signing in to the online tool? For example, can Veterans call the help number to check their dependents information by phone?

**Comment by juliepedtke** [comment]
Yeah good point, we have in the prototype "You can update your dependents online, by mail, or in person" but phone is another option!

**Comment by laurwill** [comment]
And another Q! You noted that this email is addressing Veterans who have dependents listed on either disability or pension benefits (or both). But the View or change dependents page says it's specific to disability benefits. 

Can you confirm if the tools on this page display and update dependents on both disability and pension benefits? If so, we'll need to update the static page to reflect this.

**Comment by juliepedtke** [comment]
Interesting. Revisiting the 686c (add/remove dependents), it's pretty clear about being for all three: disability, pension, and survivors so I think the static page should reflect that.  
<img width="598" alt="Screenshot 2023-09-28 at 10 11 29 AM" src="https://github.com/department-of-veterans-affairs/va.gov-team/assets/130166646/393a87a2-1334-4002-9e89-9f832d1aae6b">

**Comment by laurwill** [comment]
Thanks for confirming @juliepedtke ! It looks like the online 686c currently has "disability" in the H1 as well — so we'll need to make more global changes to the form to reflect that it affects disability, pension, and VA DIC benefits. And then we'll make the same changes to our static page. 

For now, for the email content, here are a few content notes:
- We use "verify" on VA.gov to describe a process where the Veteran must submit information — for example, verifying school enrollment info or verifying your identity for a Login.gov account. So we should use "review" here to communicate that you may not need to submit any new information if your dependents are correct.
- We updated the "Need help?" content to match current site style and provide hours. The phone number and TTY should still appear in aria code, as in the sketch version.
- Since the linked static page currently says disability only, I haven't added the specific benefit types to the email content. I'd suggest planning a follow task to update the email content once the form and page changes are live.
- I've included 2 versions here — a shorter, more evergreen version that links only to the top of the static page, and a more detailed version that links to the H2 section about adding/removing dependents. The H2 link in that version may break after the page updates, so I suggest using the shorter, more evergreen version. Happy to talk more about this!

### Version 1: Shorter with single link, more evergreen

[H1] **Review your VA dependents to avoid overpayments**

Dear [first name],
We're sending you this email to encourage you to review dependents you've added to your VA benefits. This can help you avoid debt from benefit overpayments. Tell us right away if your dependents information has changed. 

[link to https://www.va.gov/view-change-dependents/] Learn how to review or change your dependents on VA.gov 

[H2] **Need help?**
Call us at 800-827-1000 (TTY: 711). We're here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.

### Version 2: More detailed with less evergreen link to H2 section

[H1] **Review your VA dependents to avoid overpayments**

Dear [first name],
We're sending you this email to encourage you to review dependents you've added to your VA benefits. This can help you avoid debt from benefit overpayments. 

[link to https://www.va.gov/view-change-dependents/] Review your dependents on VA.gov 

We encourage you to tell us right away if your dependents information has changed. Here are some reasons you may need to add or remove dependents:
- You got married or divorced
- You had or adopted a child
- Your adult child's school enrollment changed

You can add or remove dependents online, by mail, or with the help of a trained professional.

[link to H2 section https://www.va.gov/view-change-dependents/#how-to-add-a-dependent-to-your ] Learn how to add or remove dependents on VA.gov

[H2] **Need help?**
Call us at 800-827-1000 (TTY: 711). We're here Monday through Friday, 8:00 a.m. to 9:00 p.m. ET.
